spec/unit/generator/generator_tags_spec.rb

```diff
@@ -13,6 +13,28 @@ def tags_from_strings(tags_strings)
     end
   end
 
+  describe "#detect_link_tag_time" do
+    let(:newer_tag) { nil }
+
+    let(:default_options) { GitHubChangelogGenerator::Parser.default_options.merge(verbose: false) }
+    let(:options) do
+      {
+        future_release: "2.0.0"
+      }
+    end
+    let(:generator) { described_class.new(default_options.merge(options)) }
+
+    subject do
+      generator.detect_link_tag_time(newer_tag)
+    end
+
+    before do
+      expect(Time).to receive(:new).and_return(Time.new(2020, 12, 27, 17, 0, 0, "-10:00"))
+    end
+
+    it { is_expected.to eq(["2.0.0", "2.0.0", Time.gm(2020, 12, 28, 3)]) }
+  end
```

Comment:
Thanks for adding a spec, this helps catch regressions. But, perhaps a code comment, or an "it" being described with a note that captures the essence of the change (should always use UTC since GitHub uses UTC for the things we care about, etc).

Reply:
lol, I was focusing what was being tested and the reason for this has been completely lost!

I amended my commit to add a context and an explicit comment.  Thanks!

Reply:
This looks perfect, and will guide the interested reader in the future.

Thanks, @smortex for the repairs!